library/cwm/examples/date_time.rb

```diff
@@ -0,0 +1,69 @@
+# Simple example to demonstrate object API for CWM
```

Comment:
looks like copy and paste

Reply:
It was based on the popup example so, basically copied & modified example, yes :smile: